lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -124,10 +133,9 @@
                     var validationResult = this.validateTeamData(data);
                     if (validationResult.status === false) {
                         this.showMessage(validationResult.message, validationResult.srMessage);
-                        return;
+                        return $().promise();
```

Comment:
Haha, that's the exact same link I found this morning when looking at how to fix things. I suppose we'll go with the shorter version then!